src/bootstrap.scripts.loader.js

```diff
@@ -0,0 +1,20 @@
+/* eslint func-names: 0 */
+import loaderUtils from 'loader-utils';
+
+import config from './bootstrap.config';
+import createBootstrapRequire from './utils/createBootstrapRequire';
+
+module.exports = function() {
+  if (this.cacheable) this.cacheable();
+
+  const { bootstrapPath } = loaderUtils.parseQuery(this.query);
+
+  const scriptsModules = Object.keys(config.scripts);
+
+  return (
+    scriptsModules
+      .filter(module => config.scripts[module])
```

Comment:
I was wondering if we need the filtering because if a minor release comes out with an additional file, you won't just be able to add it to your config.